regression/goto-gcc/gcc_attributes10/main.c

```diff
@@ -4,13 +4,13 @@
 #ifdef __GNUC__
 #ifndef __clang__
 
-int  x   __attribute__ ((__vector_size__ (12), __may_alias__));
+int  x   __attribute__ ((__vector_size__ (16), __may_alias__));
```

Comment:
What's the point of this change?

Reply:
To make this pass though gcc; gcc wants powers of two for the vector size.

Reply:
Uh, ok, could this please be recorded in the commit message?

Reply:
done